content/building/build-versioning.md

```diff
@@ -46,3 +46,112 @@ set -x
 cd $FCI_BUILD_DIR/ios
 agvtool new-version -all $(($BUILD_NUMBER + 1))
 ```
+
```

Comment:
I have rough plans to split this page into yaml and flutter UI to make it more clear, with specific examples, so it's good to keep a distinction of what applies to Flutter UI and yaml.